Makefile

```diff
@@ -9,7 +9,15 @@ pg_squeeze--1.4--1.5.sql pg_squeeze--1.5--1.6.sql
 DOCS = pg_squeeze.md
 
 REGRESS = squeeze
+REGRESS_OPTS = --temp-config=postgresql.conf
 
+ifdef USE_PGXS
+PG_CONFIG = pg_config
```

Comment:
`PG_CONFIG` has already been defined at the first line, no need to re-define it.